packages/amplify-e2e-tests/src/__tests__/migration/node.function.test.ts

```diff
@@ -44,7 +44,7 @@ describe('nodejs version migration tests', () => {
 
     fs.writeFileSync(projectConfigFileName, projectConfigContent, 'utf-8');
 
-    // Write back the nodejs version to 'node10.x' to test migration
+    // Write back the nodejs version to 'node12.x' to test migration
```

Comment:
Leave it on 10.x, but change `nodejs12.x` to `nodejs14.x` in line 59.

Reply:
ok changes @attilah